<?xml version="1.0" encoding="UTF-8"?>
<launch>


    <node pkg="delf" type="pidpath.py" name="pidpath" output="screen">
        <param name="linear_speed" value="0.2" />
        <param name="angular_speed" value="-0.2"/>
        <param name="goal_angle" value="90"/>
        <param name="stop_distance" value="0.5"/>
        <param name="epsilon" value="0.01"/>

    </node>


    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
        <arg name="x_pos" value="0.0"/>
        <arg name="y_pos" value="0.0"/>
        <arg name="z_pos" value="0.0"/>

    </include>

    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_gazebo_rviz.launch"/>



</launch>
